src/testing/utils.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+export const toHaveDirection = (received: Array<ReactTestInstance>) => {
+  if (received.every(instance => instance.props.style.some(style => style.flexDirection === 'row'))) {
+    return {
+      message: () =>
+        `expected received components not to have direction`,
```

Comment:
I guess the message shouldn't say the opposite if it passes as if it didn't
```suggestion
        `expected received components to have direction`,
```

Reply:
I'm quite sure its that way around. Although this is very unintuitive.

Reply:
It is correct as it is. Tested it :)